src/Intervention/Image/Imagick/Decoder.php

```diff
@@ -20,7 +20,7 @@ public function initFromPath($path)
 
             $core->setBackgroundColor(new \ImagickPixel('transparent'));
             $core->readImage($path);
-            $core->setImageType(\Imagick::IMGTYPE_TRUECOLORMATTE);
+            $core->setImageType(defined('\Imagick::IMGTYPE_TRUECOLORMATTE') ? \Imagick::IMGTYPE_TRUECOLORMATTE : \Imagick::IMGTYPE_TRUECOLOR);
```

Comment:
Shouldn't it be `IMGTYPE_TRUECOLORALPHA`...? [https://github.com/mkoppanen/imagick/blob/master/imagick_helpers.c#L1184](https://github.com/mkoppanen/imagick/blob/master/imagick_helpers.c#L1184)